pkg/controller/compliancesuite/suitererunner.go

```diff
@@ -63,6 +75,26 @@ func (r *ReconcileComplianceSuite) handleCreate(suite *compv1alpha1.ComplianceSu
 	return nil
 }
 
+// getPriorityClassName for rerunner from suite scan
+func (r *ReconcileComplianceSuite) getPriorityClassName(suite *compv1alpha1.ComplianceSuite) (string, error) {
+	// get priorityClass from suite scan
+	scans := &compv1alpha1.ComplianceScanList{}
+	scanSuiteSelector := make(map[string]string)
+	scanSuiteSelector[compv1alpha1.SuiteLabel] = suite.Name
+	listOpts := &client.ListOptions{
+		LabelSelector: labels.SelectorFromSet(scanSuiteSelector),
+		Namespace:     suite.Namespace,
+	}
+	err := r.client.List(context.TODO(), scans, listOpts)
+	if err != nil {
+		return "", fmt.Errorf("Error while getting scans for ComplianceSuite '%s', err: %s\n", suite.Name, err)
+	}
+	if len(scans.Items) == 0 {
+		return "", fmt.Errorf("No scans found for ComplianceSuite '%s'", suite.Name)
+	}
+	return scans.Items[0].Spec.PriorityClass, nil
```

Comment:
Is the `PriorityClass` attribute always present?

Reply:
It will be "", if `PriorityClass` is not set